docs/pages/more/create-expo.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: create-expo-app
+maxHeadingDepth: 4
+description: A command-line tool to create a new Expo and React Native project.
+---
+
+import { Tab, Tabs } from '~/ui/components/Tabs';
+import { Terminal } from '~/ui/components/Snippet';
+
+`create-expo-app` is a command-line tool to create and set up a new Expo and React Native project. This tool simplifies the initialization process by providing various templates to get started quickly without the need for manual configuration.
+
+## Create a new project
+
+To create a new project, run the following command:
+
+<Tabs>
+
+<Tab label="npx">
+
+<Terminal cmd={['$ npx create-expo-app@latest']} />
+
+</Tab>
+
+<Tab label="Yarn">
+
+<Terminal cmd={['$ yarn create expo-app']} />
+
+</Tab>
+
+<Tab label="pnpm">
+
+<Terminal cmd={['$ pnpm create expo-app']} />
+
+</Tab>
+
+<Tab label="bun">
+
+<Terminal cmd={['$ bun create expo']} />
+
+</Tab>
+
+</Tabs>
+
+Running the above command will prompt you to enter the app name of your project. This app name is also used in the app config's [`name`](/versions/latest/config/app/#name) property.
+
+<Terminal cmd={['What is your app named? my-app']} />
+
+## Options
+
+Uses the following options to customize the command behavior.
+
+### `--yes`
+
+Uses the default options to create a new project.
+
+### `--no-install`
+
+Skips installing npm dependencies or CocoaPods.
+
+### `--template`
+
+Running `create-expo-app` with a [Node Package Manager](#node-package-managers-support) initializes and sets up a new Expo project using the default template.
+
+You can use the `--template` option to choose one of the following templates:
+
+| Template                | Description                                                                                                                                                                           |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Default                 | Default template. Designed to build multi-screen apps. Includes recommended tools such as Expo CLI, Expo Router library and TypeScript configuration enabled. Suitable for most apps. |
```

Comment:
We can consider adding links to source of templates that we keep on GH. It could be a separate column, or we can link the name:
* https://github.com/expo/expo/tree/main/templates